slakh96/no-mans-land
# Update to new OxygenMeter

Testing
- Delete previous OxygenMeterCanvas if it exists in your scene
- Drag and drop ONLY OxygenCanvas from Assets/Prefabs/Oxygen to scene.
- Should work as normal.

![Screen Shot 2021-10-29 at 10 33 55 PM](https://user-images.githubusercontent.com/32211479/139517472-38d08bde-8f6a-41d3-8c1f-6cc821a2ea3f.png)
